src/components/EditorBottomBar.tsx

```diff
@@ -1,15 +1,32 @@
 import "./EditorBottomBar.css";
 import { FormattedMessage } from "react-intl";
+import { useAppServices } from "./AppServicesContext";
 
-interface EditorBottomBarProps {}
+interface EditorBottomBarProps {
+  nextSrc: string;
+}
+
+export const EditorBottomBar = ({
+  nextSrc,
+  ...otherProps
+}: EditorBottomBarProps) => {
+  const {
+    window: { location },
+  } = useAppServices();
+  const onContinue = (): any => {
+    location.href = nextSrc;
+  };
 
-export const EditorBottomBar = ({ ...otherProps }: EditorBottomBarProps) => {
   return (
     <div className="ed-cta-footer" {...otherProps}>
       <button type="button" className="dp-button button-medium secondary-green">
         <FormattedMessage id="exit_edit_later" />
       </button>
-      <button type="button" className="dp-button button-medium primary-green">
+      <button
+        type="button"
+        className="dp-button button-medium primary-green"
+        onClick={onContinue}
+      >
```

Comment:
If the URL is known, we should use a link (`<a>`) in place of a button.

Reply:
Applied